src/Sarif.FunctionalTests/v2/ConverterTestData/FxCop/Readme.md

```diff
@@ -0,0 +1,15 @@
+Notes on producing test examples.
+
+This output is an FxCop run against a debug build of SARIF-SDK 
+
+0. Enlist in the SARIF-SDK. You will update the test files in this version.
+1. Git clone into a second version of the tree, e.g.:
+   git clone https://github.com/microsoft/sarif-sdk sarif-sdk-fxcop-test
```

Comment:
>git [](start = 3, length = 3)

Nit: Since this is a Markdown file, consider enclosing command lines in backticks.